modules/e2ee/Context.js

```diff
@@ -98,6 +116,52 @@ export class Context {
     _setKeys(keys) {
         this._cryptoKeyRing[this._currentKeyIndex] = keys;
         this._sendCount = BigInt(0); // eslint-disable-line new-cap
+        this._lastSignatureAt.clear();
```

Comment:
just to check: if we ratchet because someone joins, we use setKey from the client code which resets calls _setKeys and therefore we reset.
This calls for another test :-)

Reply:
test added :-)